docs/prod-maintain-secure-upgrade.md

```diff
@@ -4,11 +4,13 @@ This page covers topics that will help you maintain a healthy, up-to-date, and
 secure Zulip installation, including:
 
 - [Upgrading](#upgrading)
-- [Deploying from a git repository](#deploying-from-a-git-repository)
+- [Upgrade from a git repository](#upgrade-from-a-git-repository)
 - [Backing up and restoring](#backing-up-and-restoring)
```

Comment:
Needs to be `[Back up and restore](#back-up-and-restore)` to match changed heading.